docs/using/docker-image.rst

```diff
@@ -0,0 +1,167 @@
+.. Copyright 2016 tsuru authors. All rights reserved.
+   Use of this source code is governed by a BSD-style
+   license that can be found in the LICENSE file.
+
++++++++++++++++++++++++++++++++++++
+Deploying Docker Image applications
++++++++++++++++++++++++++++++++++++
+
+Overview
+========
+
+This document is a hands-on guide to deploying a simple Docker Image web application.
+
+Creating the app within tsuru
+=============================
+
+To create an app, you use the command `app-create`:
+
+.. highlight:: bash
+
+::
+
+    $ tsuru app-create <app-name> <app-platform>
+
+For Docker Images, not exists one specific platform, but we can use ``static``! Let's be over creative and develop a hello
+world tutorial-app, let's call it "helloworld":
+
+.. highlight:: bash
+
+::
+
+    $ tsuru app-create helloworld static
+
+To list all available platforms, use the command `platform-list`.
+
+You can see all your applications using the command  `app-list`:
+
+.. highlight:: bash
+
+::
+
+    $ tsuru app-list
+    +-------------+-------------------------+--------------------------------+
+    | Application | Units State Summary     | Address                        |
+    +-------------+-------------------------+--------------------------------+
+    | helloworld  | 0 of 0 units in-service | helloworld.192.168.50.4.nip.io |
+    +-------------+-------------------------+--------------------------------+
+
+Application code
+================
+
+A simples `Dockerfile`:
+
+.. highligth:: Dockerfile
+
+::
+
+    FROM golang
+    RUN mkdir /app
+    WORKDIR /app
+    ADD . /app/
+    RUN go build .
+    ENTRYPOINT ./app
+
+A simple web application in Go `main.go`:
+
+.. highlight:: go
+
+::
+
+    package main
+
+    import (
+        "fmt"
+        "net/http"
+        "os"
+    )
+
+    func main() {
+        c := make(chan os.Signal, 1)
+        signal.Notify(c, os.Interrupt)
+        go func(){
+            for sig := range c {
+                if sig == os.Interrupt || sig == os.Kill {
+                    os.Exit(1)
+                }
+            }
+        }()
+        http.HandleFunc("/", hello)
+        fmt.Println("running on "+os.Getenv("PORT"))
+        http.ListenAndServe(":"+os.Getenv("PORT"), nil)
+    }
+
+    func hello(res http.ResponseWriter, req *http.Request) {
+        fmt.Fprintln(res, "hello, world!")
+    }
+
+
+Buildind the image
+==================
+
+.. highlight:: bash
+
+::
+
+    docker login registry.myserver.com
+
+    docker build -t registry.myserver.com/image-name .
+
+
+Don't forgot the dot(.) at the end of the command, this indicates where the Dockerfile stay
```

Comment:
I think it should be `forget` instead of `forgot`. I would also change `stay` for `is placed`